at least one qa is based on summarizing answers from a single document that has 2 small paragraphs and the answer is useful

d-geula opened the pull request:
It isn't much, but I wanted to at least create this draft PR so you can tell me whether I did the mocking/patching correctly. Sorry for the messy commits, and I'll pick up the pace over the next few days.

btw I also wanted to ask how I should title a PR like this. Currently it's just using the branch name but it seems pretty long and messy to me. Just curious.

d-geula committed:
temp

d-geula committed:
Refactor 'test_sql_chain' to use patch instead of passing 'llm' as argument

d-geula committed:
Delete docs_summary.ipynb

forgot to remove this.

d-geula committed:
Update from main

Also forgot to delete `extra_context`. It was only for testing purposes.

d-geula committed:
Remove unnecessary print statement

itaifrenkel commented:
I understand your question about the title. You could edit the pr title  and make it shorter.it’s a descent kpi though, so I wouldn’t change the parent task name

d-geula committed:
.

d-geula committed:
Merge branch 'main' into 13-at-least-one-qa-is-based-on-summarizing-answers-from-a-single-document-that-has-2-small-paragraphs-and-the-answer-is-useful

d-geula committed:
Update enhanced_summary.py

d-geula commented:
Ok, I think it's how you wanted it now. I moved the caching-related code to `enhanced_summary` and updated from main to remove unnecessary changes. There are still a couple of small changes to other files which I could move to their own branch but I feel like that's just overcomplicating things.
